programs/wen-new-standard/src/instructions/mint/royalties/add.rs

```diff
@@ -78,6 +74,12 @@ impl<'info> AddRoyalties<'info> {
 }
 
 pub fn handler(ctx: Context<AddRoyalties>, args: AddRoyaltiesArgs) -> Result<()> {
+    // validate that the fee_basis_point is less than 10000 (100%)
+    require!(
+        args.royalty_basis_points <= 10000,
```

Comment:
why do you need royalties to be LEQ 100%?

Reply:
I'm not sure, but I guess it might break the payment of royalties. How will the program behave if it see something to pay more than 100% of the cost? Just added that sanity check to make sure we handled that edge case

Reply:
Well it would just transfer that amount, it doesn't make a difference if it's more than the cost. Seems like a weird opinionation to make. What's the basis for restricting royalties to be less than or equal to the purchase price? You could have a situation where the royalty is 200% and for a 1 SOL purchase the artist gets 2 SOL. Is it just for convenience of calculation? Just want to make sure this isn't just a constraint we're imposing and is expected more broadly.

Reply:
Yeah sorry, i just added a sanity check cause i was thinking that the cap should be 100%. If not we can delete it, no problem